.github/workflows/notebook-test-suite.yml

```diff
@@ -0,0 +1,143 @@
+# This workflow runs Jupyter notebook based regression test suites.
+# It can be run manually run on github Actions or invoked through
+# the github Action workflow api.
```

Comment:
Can it?  Is it a permission thing or an understanding thing that I can't do that?

Reply:
Good point. I lied. I took out the workflow dispatch. I am going to add it back.

Reply:
Ok. Re-enabled workflow dispatch. Now you should be able to run manually if you have permission.